app/scripts/migrations/076.js

```diff
@@ -18,28 +15,16 @@ export default {
 };
 
 function transformState(state) {
-  if (state.CollectiblesController) {
-    const {
-      allCollectibleContracts,
-      allCollectibles,
-      ignoredCollectibles,
-      ...remainingState
-    } = state.CollectiblesController;
+  if (state.NftsController) {
```

Comment:
The migration needs to remain unaltered - we need to continue migrating from the old controller to the new one